resqpy/well/well_object_funcs.py

```diff
@@ -533,7 +535,10 @@ def lookup_from_cellio(line, model):
 
     # Check if a StringLookupTable already exists in the model, with the same name and values
     for existing in model.parts_list_of_type('obj_StringTableLookup'):
-        table = rqp.StringLookup(parent_model = model, root_node = model.root_for_part(existing))
+        table = rqp.StringLookup(
+            parent_model = model,
+            uuid = model.uuid_for_part(part_name = existing))  #TODO: confirm it is okay to change root_node to uuid
```

Comment:
Confirmed that use of uuid is much preferable to root_node.

I would recommend changing the for statement to:
`for existing_uuid in model.uuids(obj_type = 'StringTableLookup'):`